otel-extension/src/main/java/akka/actor/impl/LocalActorRefProviderAdvice.java

```diff
@@ -1,12 +1,15 @@
 package akka.actor.impl;
 
-import io.scalac.mesmer.otelextension.instrumentations.akka.actor.InstrumentsProvider;
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+import io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleEvents;
 import net.bytebuddy.asm.Advice;
 
 public class LocalActorRefProviderAdvice {
 
   @Advice.OnMethodExit
-  public static void actorOfExit() {
-    InstrumentsProvider.instance().actorsCreated().add(1);
+  public static void actorOfExit(
+      @Advice.Argument(0) ActorSystem system, @Advice.Return ActorRef ref) {
+    system.eventStream().publish(new ActorLifecycleEvents.ActorCreated(ref));
```

Comment:
Hi @lgajowy, I am curious what is the purpose on use EventStream instead of using Otel meter to measure it?

Reply:
We ultimately are using the otel meter. It's just that termination emits an akka Terminated() event so we need to intercept it somehow. We try to keep such event processing to the minimum now and rely on pure otel instruments but it's not always possible I think :)

Reply:
Yep. I think I got the point, rather than enhance another method like `ActorCell.stop()`, watching an Actor Lifecycle is the better option.

Thanks for your time in explaining.